scripts/arm32_ci_script.sh

```diff
@@ -203,9 +264,17 @@ do
             exit_with_error "--buildConfig can be only Debug or Release" true
         fi
         ;;
+    --arm)
+        __ARMRootfsImageBase="rootfs-u1404.ext4"
```

Comment:
Nothing is decided for testing yet and I just keep previous code. Do you have any idea for testing ?
I think we also can make use of Docker for testing too.